specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/Diagnostics.json

```diff
@@ -432,6 +479,218 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/detectorProperties/daprApi/": {
+      "get": {
+        "tags": [
+          "ManagedEnvironments",
+          "Diagnostics"
+        ],
+        "summary": "Get the Dapr components of a Managed Environment.",
+        "description": "Get the Dapr components of a Managed Environment used to host container apps.",
+        "operationId": "ManagedEnvironmentsDiagnostics_ListDaprComponents",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprComponentsCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Dapr Components": {
+            "$ref": "./examples/Diagnostics_DaprComponents_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/detectorProperties/daprApi/{componentName}": {
```

Comment:
>/{ [](http://example.com/codeflow?start=19&length=2)

needs breaking changes review

Reply:
It turns out that 2022-06-01-preview isn't the right api version to use as of now. I am going to create a new PR and close this PR.